slick/src/sphinx/migrations.rst

```diff
@@ -26,3 +26,16 @@ slick-migration-api
 library for defining database migrations, for use with Slick. It supports
 altering database schemas using a type-safe Scala DSL. Unfortunately, it
 does not support Slick of version 3.0 or later for the time being.
+
+Flyway
+--------------------------
+
+`Flyway <https://flywaydb.org/>`_ is a database migration tool that can
+ run SQL or Java against a database.
```

Comment:
Hard to spot... but there's an extra space at the start of line 34, which causes "run SQL..." to be indented :-)

![database migrations slick 3 2 0-snapshot documentation 2016-05-31 12-21-45](https://cloud.githubusercontent.com/assets/102661/15672740/5f18defc-272a-11e6-97ff-3419cad37f91.png)

Can we remove that space please?

BTW, you can generate the site with `sbt make-site`. The HTML is written to _slick/slick/target/sphinx/docs/index.html_.